diffrax/custom_types.py

```diff
@@ -131,3 +133,10 @@ def __class_getitem__(cls, item):
 DenseInfo = Dict[str, PyTree[Array]]
 DenseInfos = Dict[str, PyTree[Array["times", ...]]]  # noqa: F821
 sentinel: Any = eqxi.doc_repr(object(), "sentinel")
+
+
+class LevyVal(eqx.Module):
+    h: Scalar = field(default=0.0)
+    W: PyTree[Array] = field(default=None)
+    J: Optional[PyTree[Array]] = field(default=None)
+    H: Optional[PyTree[Array]] = field(default=None)
```

Comment:
`field` isn't necessary if you're only specifying default values. Just set them as the default directly.

That said, are default values even needed here? I like to say that "default values are the root of unexpected behaviour", and as this is a type that is only instantiated internally, we can always just pass in every value and I think that shouldn't be too much work?

Reply:
That's fair. The main idea was that as new types of Levy area are added, there is no need to change all the code that uses LevyVals. But then again, that's pretty minor, so I can easily get rid of these defaults if you think that's the way to go.